paddle/operators/adadelta_op.cc

```diff
@@ -84,22 +83,21 @@ class AdadeltaOpMaker : public framework::OpProtoAndCheckerMaker {
                    "numerical stability")
         .SetDefault(1.0e-6f);
     AddComment(R"DOC(
-Adadelta Updates Operator.
+Adadelta Optimizer.
 
-This implements the Adadelta optimizer[1]. Adadelta is a per-dimension
-adaptive learning rate method for gradient descent.
+We implement the Adadelta optimizer as explained in:
```

Comment:
Do you think we should use the first person here?

Reply:
Fixed. Thanks for the comment.